airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -1,7 +1,6 @@
 import { Navigate } from "react-router-dom";
 
 import { RoutePaths } from "../../../pages/routePaths";
-import { CloudRoutes } from "../cloudRoutePaths";
```

Comment:
Can the `CloudRoutes.SelectWorkspace` be deleted from its file now? Seems we didn't do that in this PR.

Reply:
You mean removing `SelectWorkspace` from the `CloudRoutes` enum entirely? Yes, we can do that - addressed here: 47a1e15174336a27f43065ffebf7e965b3415c8f